File: TD/TD3/Code/4.cpp
/* emander à USER une valeur entre 0 et 20, la retourner après */
#include <iostream>

int retour()
{
  int a;
  
  do {
    std::cout << "Donne moi une valeur entre 0 et 20 : " << std::flush;
    std::cin >> a;
  } while ((a < 1) || (a > 20));
  
  return a;
}

int main()
{
  int a;
  
  a = retour();
  
  std::cout << std::endl;
  
  return 0;
}

File: TD/TD3/Code/Python/4.py
#!/usr/bin/python
# Demander à USER une valeur entre 0 et 20, la retourner après


def retour():
    print('Donne moi une valeur entre 0 et 20 : ', end='')
    val = int(input())

    while val < 0 or val > 20:
        print('La valeur doit être entre 0 et 20. Recommence : ', end='')
        val = int(input())

    return val


retour()
